internal/resources/resource_host_test.go

```diff
@@ -29,23 +29,23 @@ variable "location" {
 	# default = "USA:West Central:FTC DEV 4"  
 	default = "USA:Texas:AUSL2"
 }
-data "quake_available_resources" "compute" {
+data "metal_available_resources" "compute" {
 	
 }
-resource "quake_host" "test_host" {
+resource "metal_host" "test_host" {
```

Comment:
need to replace 'quake' with 'hpegl_metal' prefix everywhere.

Reply:
Done